partner_firstname/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     "name": "Partner first name and last name",
     "summary": "Split first name and last name for non company partners",
-    "version": "17.0.1.0.3",
+    "version": "17.0.1.0.4",
```

Comment:
Don't update this. It will be by ocabot

Reply:
Thanks! I thought that I had to update it so odoo.sh knows it has to update this module to apply the changes :)
I'm reverting that

Reply:
done